src/test/java/tests/AgodaHotelSearchTest.java

```diff
@@ -30,26 +43,11 @@ void setup() {
     @Test(description = "Search and sort hotel successfully")
     void TC01_Search_Sort_Hotel() {
 
-        String place = "Da Nang";
-        int targetRooms = 2;
-        int targetAdults = 4;
-        String dateFormat = "yyyy-MM-dd";
-
-        String checkInDate = DateTimeUtils.getNextFriday(dateFormat); // Next Friday
-        String checkOutDate = DateTimeUtils.getDateFromSpecificDate(checkInDate, 3, dateFormat); // 3 days from next Friday
-
-        step("Step: Searching for: " + place +
-                ", Check-in: " + checkInDate +
-                ", Check-out: " + checkOutDate +
-                ", Rooms: " + targetRooms +
-                ", Adults: " + targetAdults);
-
-        agodaHomePage
-                .enterDestination(place)
-                .selectDates(checkInDate, checkOutDate)
-                .setFamilyTravelers(targetRooms, targetAdults);
+        step("Select VND Currency");
+        agodaHomePage.selectCurrency("Vietnamese Dong");
 
-        agodaSearchResultsPage = agodaHomePage.clickSearchButton();
+        step("Step 1: Searching hotel");
```

Comment:
same here, if you're already logged the step in the method then delete this line
![image](https://github.com/user-attachments/assets/e7fe8292-0857-46d8-8b10-4a44424bfb56)